include/deal.II/dofs/dof_tools.h

```diff
@@ -1452,6 +1452,18 @@ namespace DoFTools
   std::vector<IndexSet>
   locally_relevant_dofs_per_subdomain (const DH   &dof_handler);
 
+
+  /**
+   * Same as extract_locally_relevant_dofs() but for multigrid DoFs
+   * for the given @p level.
+   */
+  template <class DH>
+  void
+  extract_locally_relevant_mg_dofs (const DH &dof_handler,
+                                    const unsigned int level,
```

Comment:
Should we decide, whether we use* _mg_* or *_level_* in the names of such functions?